docs/get_started/assembly_line_setup.md

```diff
@@ -1,90 +0,0 @@
----
-id: assembly_line_setup
-title: |
-  Set up your Document Assembly Line
-sidebar_label: |
-  Set up your Document Assembly Line
-slug: /get_started/assembly_line_setup
----
-
-<!-- TODO: Add link to someplace to give feedback or some kind of forum. -->
-
-The guidance below is for organizations that want to set up their **own** copy of the Assembly Line, not for individual automators.
-
-How do you develop and deploy court forms effectively? This is what has worked
-for us. Adjust it when you need to. We'd love to hear about the changes you make
-for your organization!
-
-## Setting up your organization
-
-This happens once in the lifetime of an organization. These steps will be filled out as we develop this documentation further. Talk to us to learn more.
-
-1. [Create a docassemble server](https://suffolklitlab.org/legal-tech-class/docs/practical-guide-docassemble/setup-server)
-1. [Add the AssemblyLine package to your server](installation.md)
-1. [Copy](https://help.trello.com/article/802-copying-cards-lists-or-boards) the Trello [checklist boards](#trello-boards)
-
-You should also consider:
-
-1. [enabling GitHub Playground integration](https://docassemble.org/docs/config.html#github)
-2. inviting each developer who will use your server to [create their own accounts](https://docassemble.org/docs/users.html#add)
```

Comment:
This page was almost entirely geared towards replicating the Document Assembly Line concept of 2022, which as far as I can tell isn't how anyone is thinking about it any more (i.e., dozens or hundreds of people making small contributions regardless of their skill level).

The rest of it can be found elsewhere. For example, the GitHub documentation does a better job of documenting GitHub orgs, and the Docassemble documentation does a better job of documenting GitHub integrations and Docassemble server setup and configuration.

Reply:
> as far as I can tell isn't how anyone is thinking about it any more (i.e., dozens or hundreds of people making small contributions regardless of their skill level).

True, but I think a lot of our partners still use orgs. Orgs aren't just for hundreds of people. It doesn't need to be much. For example, it might be worth mentioning GitHub orgs with a link to the GitHub docs and maybe the docassemble docs.